addons/website_slides_survey/tests/test_course_certification_failure.py

```diff
@@ -85,23 +85,27 @@ def test_course_certification_failure_flow(self):
         self.fill_in_answer(retry_user_input, certification.question_ids)
         # forces recompute of partner_ids as we delete directly in relation
         self.channel.invalidate_model()
-        self.assertNotIn(self.user_public.partner_id, self.channel.partner_ids, 'Public user should have been kicked out of the course because they failed their last attempt')
 
+        channel_partner = self.env['slide.channel.partner'].with_context(active_test=False).search([
+            ('channel_id', 'in', self.channel.ids),
+            ('partner_id', 'in', slide_partner.partner_id.ids)
```

Comment:
for better git history if someone add a line after
```suggestion
            ('partner_id', 'in', slide_partner.partner_id.ids),
```